include/circt/Dialect/Arc/ArcPasses.h

```diff
@@ -46,6 +46,7 @@ std::unique_ptr<mlir::Pass> createLowerVectorizationsPass(
 std::unique_ptr<mlir::Pass> createMakeTablesPass();
 std::unique_ptr<mlir::Pass> createMuxToControlFlowPass();
 std::unique_ptr<mlir::Pass> createSimplifyVariadicOpsPass();
+std::unique_ptr<mlir::Pass> createSplitFuncsPass(unsigned splitBound = 20000);
```

Comment:
I think you should be able to use the options struct that table-gen produces for you. That way you don't have to redefine the default split bound 🙂 
```suggestion
std::unique_ptr<mlir::Pass> createSplitFuncsPass(const SplitFuncsOptions &options = {});
```

Reply:
Very nifty, the redefinition was bugging me! Thanks!